KS-최경수/3주차 미션/pratice2/src/components/Items.jsx

```diff
@@ -0,0 +1,12 @@
+const Items = ({id, data}) => {
+
+  const movieimgPath = "https://image.tmdb.org/t/p/original" + data.poster_path;
+  console.log(movieimgPath);
```

Comment:
```suggestion
  const movieimgPath = `https://image.tmdb.org/t/p/original${data.poster_path}`
```

이런 식으로 비교적 간단하게 변수가 들어오는 문자열을 관리할 수 있답니다!